pkg/api/v1beta2/types.go

```diff
@@ -1205,8 +1205,10 @@ type Minion struct {
 	Status NodeStatus `json:"status,omitempty" description:"current status of node"`
 	// Labels for the node
 	Labels map[string]string `json:"labels,omitempty" description:"map of string keys and values that can be used to organize and categorize minions; labels of a minion assigned by the scheduler must match the scheduled pod's nodeSelector"`
-	// External ID of the node
+	// External ID of the node (obsolete)
```

Comment:
As discussed via IRC, please use "deprecated" and put in the description tag, as in:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L671

Here and all other API versions.